tests/kuttl/tests/basic-deployment/04-assert-service-cert-rotation.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+  - script: |
+      echo "Waiting for OpenStack control plane to be ready..."
+      oc wait openstackcontrolplane -n $NAMESPACE --for=condition=Ready --timeout=400s -l core.openstack.org/openstackcontrolplane
+
+  - script: |
+      echo "Get fingerprints of all service certs"
+      echo "PWD $PWD"
+      oc exec -i openstackclient -n $NAMESPACE -- bash -s < tests/kuttl/common/osp_endpoint_finger_prints.sh > /tmp/endpoint_fingerprints_after
```

Comment:
the issuer is here. the base of the TestAssert is the install_yamls dir, not like in TestStep the kuttl test dir

```
    logger.go:42: 08:39:03 | basic-deployment/4-rotate-service-certs | PWD /alabama/install_yamls
    logger.go:42: 08:39:03 | basic-deployment/4-rotate-service-certs | sh: line 3: tests/kuttl/common/osp_endpoint_finger_prints.sh: No such file or directory
```